app/controllers/articles_controller.rb

```diff
@@ -37,12 +51,12 @@ def update
   private
 
   def add_categories_to_article
-      categories = params[:article][:categories]
-      categories.each do |category_id|
-        category = Category.find_by(id: category_id)
-        @article.categories.include?(category) ? next : @article.categories << category
-      end
+    categories = params[:article][:categories]
+    categories.each do |category_id|
+      category = Category.find_by(id: category_id)
+      @article.categories.include?(category) ? next : @article.categories << category
     end
+  end
```

Comment:
This is why thoroughly performed code reviews are important. By merging badly formatted code in a previous PR and fixing it now, you are clouting this PR and making it harder to review. Please set higher standards for the team.

Reply:
We will do better, thanks for pointing this out!

Reply:
Okay, thx.